scripts/superdesk-packaging/packaging.js

```diff
@@ -12,11 +12,12 @@
 
     'use strict';
 
-    PackagesService.$inject = ['api', '$q', 'archiveService', 'lock', 'autosave', 'authoring', 'desks'];
-    function PackagesService(api, $q, archiveService, lock, autosave, authoring, desks) {
+    PackagesService.$inject = ['api', '$q', 'archiveService', 'lock', 'autosave', 'authoring', 'authoringWorkspace', 'desks', '$rootScope'];
+    function PackagesService(api, $q, archiveService, lock, autosave, authoring, authoringWorkspace, desks, $rootScope) {
         var self = this;
 
         this.groupList = ['main', 'story', 'sidebars', 'fact box'];
+        this.packageGroupItems = {};
```

Comment:
Ok, let me make the changes. I also have to fix the test as well, then I'll send you back the updated pull request for a review